### DISCUSSION TITLE
Connecticut Planning Regions

### DISCUSSION THREAD

**Discussion by wadejes1** [pdil/usmap#112] [Ideas]
Hello, I recent found this package on R and it has been very useful for making state level maps. I am starting to use the county function a bit more. I mainly use the Census Bureaus ACS data and Connecticut recently got rid of the county structure, instead of 8 Counties there are [9 planning regions ](https://www.federalregister.gov/documents/2022/06/06/2022-12063/change-to-county-equivalents-in-the-state-of-connecticut). This creates issues when using the latest ACS data because the data is released at a planning region level. The counties now show up as grey because the FIPs no longer match with the County fips the mapping program uses. Any chance there is a future update in the works that could account for this change? I really enjoy using this package on R, makes like much easier. 
![image](https://github.com/user-attachments/assets/e5b5b2e7-c3f5-49af-82f6-5ccf1570bda3)

**Comment by pdil** [comment]
Yeah I plan on adding the ability to do this (as reported in #110) soon. I'm still working out the best way to do this without introducing confusion or extra complexity.

**Comment by pdil** [comment]
This work is finally complete and available in the newly released [usmap v0.8.0](https://github.com/pdil/usmap/releases/tag/v0.8.0). The update introduces the `data_year` parameter for most functions and includes data from the years 2021-2024 (inclusive) to choose from (the latest is used by default).
